draft-rha-jose-hpke-encrypt.md

```diff
@@ -306,6 +345,47 @@ In the above example, the JWE Protected Header value is:
 }
 ~~~
 
+~~~
+{
+  "protected": "eyJlbmMiOiJBMTI4R0NNIiwiZXBrIjp7Imt0eSI6IkVLIiwiZWsiOiJCUFJUS244bVFMNGhOMWF5b2tSOGdrcFR5NUhRbGQ0TjBIWFhCOWNYdGpVSVEzN3pzSkRMN1R1Z1ZrbUQxYUZZVHgtMGJNMHRmeHplakxjdFNES2pNUXMifX0",
+  "encrypted_key": "zR0ArfrVVRQ9-X_heDU2riwx36QxLBffRrKAWU-tLC4",
+  "iv": "o3v11Hw6gUxUN-pY",
+  "ciphertext": "Ny-2IDGHMI3MzVsUAVMGNoKAZfoewTZ1dkAIBikPy4eZUfHW_LPhhKpD6Mf4zYGkhAeLwGgJKjyDoFIj0EuDsEtJ",
+  "tag": "0sfzHJvxVoWt02EPxMTh8w"
+}
+~~~
+{: #key-encryption-single-recipient-general-json align="left" title="Key Encryption (single recipient) General JSON Serialization"}
+
+In the above example, the JWE Protected Header value is: 
+
+~~~
+{
+  "enc": "A128GCM",
+  "epk": {
+    "kty": "EK",
+    "ek": "BPRTKn8mQL4hN1ayokR8gkpTy5HQld4N0HXXB9cXtjUIQ37zsJDL7TugVkmD1aFYTx-0bM0tfxzejLctSDKjMQs"
+  }
+}
+~~~
+
+~~~
+eyJlbmMiOiJBMTI4R0NNIiwiZXBrIjp7Imt0eSI6IkVLIiwiZWsiOiJCSjdyZE5ia3Z3dW5zc2RqdTVXRGtBYXpMYUJYM0lkY0xSankxUkRTQTlzaWowMGpkeWJhSHVRUFR0NlAzMUJpMG5FMmtZV183TF90YUFxQUZLNzVEZWsifX0.xaAa0nFxNJxsQQ5J6EFdzUYROd2aV517o2kZnfwhO7s.AgBYEWTj-EMji17I.Ejwu2iEP4xs3FfGO_zTZYu35glQmUvd_qpHpvB1hNqg6Yz5ek3NsZRGMzd--HYWvABNslxBkRwrkZDXnv_BTgOTj.u0ac86ipoAwUZuYwkaKwNw
+~~~
+{: #key-encryption-single-recipient-compact align="left" title="Key Encryption (single recipient) Compact"}
+
+In the above example, the JWE Protected Header value is: 
```

Comment:
The example is repeated twice.

Reply:
its intentional to repeat the decoding of the protected headers, in the case of JSON and compact for the same message they have to be the same.

Reply:
Got it.